windowslib/src/main/resources/scala-native/windows/time.c

```diff
@@ -0,0 +1,133 @@
+// Windows specific copy of posixlib/time.h
+// Uses *_s variants of methods instead of *_r
+#if defined(_WIN32)
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
I really think we should consider just doing a `clib` time API - We should have virtually all the time stuff in `clib` and then just the posix specific in the `posixlib`.

Both time APIs can access the same C code and we could just deprecate most of the Scala in `posixlib`. You could then add just windows specific stuff in another interface as needed.

Do you think we should just address that in another PR and try it get it merged first?